### PR TITLE
enh: Adding verticle timeline to session page

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #
Screenshot from 2020-08-31 22-43-56

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
There shouldn't be any line breaks in time. See the example URL in meeting notes

**Comment by maze-runnar** [comment]
Screenshot from 2020-09-01 20-42-48

**Comment by iamareebjamal** [comment]
Now it's 12 hour time without AM/PM

**Comment by maze-runnar** [comment]
@iamareebjamal now it's showing 24 hour format, similar to website generate;
Screenshot from 2020-09-01 22-36-33

**Comment by iamareebjamal** [comment]
@maze-runnar The build is failing. Please finalize old issues before taking on new ones

**Comment by maze-runnar** [comment]
> @maze-runnar The build is failing. Please finalize old issues before taking on new ones

Done

**Comment by iamareebjamal** [comment]
Thanks. I'll review it

File: app/templates/public/sessions.hbs [hunk 1]
```diff
@@ -44,12 +44,23 @@
   </InfinityLoader>
 {{/if}}
 
-<div class="mt-8">
+
+<div class="history-tl-container">
+  <ul class="tl">
   {{#each this.model.session as |session|}}
-    <Public::SessionItem @session={{session}} @timezone={{this.timezone}} />
+    <li class="tl-item" ng-repeat="item in retailer_history">
```
Comment (iamareebjamal) [inline app/templates/public/sessions.hbs:51]: Unnecessary code

Comment (maze-runnar) [reply app/templates/public/sessions.hbs]: @iamareebjamal I have removed unrequired code from css and hbs both.

**Comment by mariobehling** [comment]
We can improve the design at a later stage in detail, but for now at least the times should follow the spacing as the above design items. Please move the time to the right in order to follow an imaginary line from top to bottom (based on the headings above). Thanks.

91869617-bc329280-ec93-11ea-9293-92b979d685e9

**Comment by iamareebjamal** [comment]
@maze-runnar Please center align the time so that it looks better

**Comment by maze-runnar** [comment]
Screenshot from 2020-09-07 18-38-58

**Comment by maze-runnar** [comment]
Screenshot from 2020-09-07 18-42-11

**Comment by iamareebjamal** [comment]
Nothing has changed. Please reread the comments and picture

**Comment by iamareebjamal** [comment]
> at least the times should follow the spacing as the above design items. Please move the time to the right in order to follow an imaginary line from top to bottom (based on the headings above). Thanks.
> 
> 91869617-bc329280-ec93-11ea-9293-92b979d685e9

**Comment by iamareebjamal** [comment]
The time now seems even more left aligned. Even going as far as being aligned to the sidebar

**Comment by maze-runnar** [comment]
> Nothing has changed. Please reread the comments and picture

the line should be in left & time should be in right?

**Comment by iamareebjamal** [comment]
Time should not be aligned to the left. See mario's image

**Comment by maze-runnar** [comment]
> Time should not be aligned to the left. See mario's image

I am not exactly getting his image, what is trying to say.

**Comment by iamareebjamal** [comment]
The time is aligned to the left. It should move to the right

**Comment by iamareebjamal** [comment]
Not aligned to the right. It should move slightly to the right to match the margin on the rest of the content

**Comment by iamareebjamal** [comment]
@snitin315 Can you please explain if you understand?

**Comment by maze-runnar** [comment]
> Not aligned to the right. It should move slightly to the right to match the margin on the rest of the content

ok, got it

**Comment by snitin315** [comment]
Expected- 

```
    Session
    10-11
    11-12
```
Current-

```
    Session
  10-11
  11-12
```

**Comment by maze-runnar** [comment]
I have deleted the fork, and fork again the project, so creating a new PR for this

**Pull request closed without merging**